functions/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
please change this to the following:
`node_modules/`

did you run the remove command?